src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,165 @@
 package org.example;
 
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
+
+    private static String[][] tasks;
+    private static Scanner scanner;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
```

Comment:
CLEAN-CODE: prosim u konstanty OPTIONS pridej taky ze je pristupna jen ve tride Main pridanim private. O objektove orientovanem programovani se budeme vice ucit v lednu.

```suggestion
    private static final String[] OPTIONS = {"add", "remove", "list", "exit"};
```

Reply:
Opraveno.